### PR TITLE
start of variable reward curves

### PR DESCRIPTION
**Pull request by thehannahlerman**

### CODE REVIEW

File: app/routes/app+/showcase.tsx [hunk 1]
```diff
@@ -228,30 +226,30 @@ function SubmissionsCard({ submissions, projects }: { submissions: CombinedDoc[]
   return (
     <div className="space-y-4">
       {submissions.map((s) => {
+        const averageScore = (s.score?.reviewSum || 0) / (s.score?.reviewCount || 0);
```
Comment (termaatFlipside) [inline app/routes/app+/showcase.tsx:229]: divide by 0 issue?

Comment (thehannahlerman) [reply app/routes/app+/showcase.tsx:229]: oops yup, fixed this in most places but must have missed this one, thank you

File: indexer/worker.ts [hunk 2]
```diff
@@ -59,9 +60,9 @@ worker.onEvent(LaborMarket, "RequestSignal", async (event) => {
   return indexerRequestSignalEvent(event);
 });
 
-worker.onEvent(LaborMarket, "RequestReviewed", async (event) => {
-  return indexerRequestReviewedEvent(event);
-});
+// worker.onEvent(LaborMarket, "RequestReviewed", async (event) => {
```
Comment (sebruiz) [inline indexer/worker.ts:63]: I think this can be restored

**Review by sebruiz — APPROVED** [review]

**Review by sebruiz — CHANGES_REQUESTED** [review]
I was testing to see if I could make a new LM this morning and couldn't. Might be related to the dropdown with the curve selection?

**Pull request merged**